Server/ReasnAPI/ReasnAPI/Models/DTOs/UserInterestDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ReasnAPI.Models.DTOs
+{
+    public class UserInterestDto
+    {
+        public int InterestId { get; set; }

```

Comment:
I think we can already do here InterestDto itself instead of Id, it's only a helper list for a user, so I think it will be easier to manipulate then. Like in event - there is no List of TagIds or ParameterIds but of specific objects